app/controllers/admin/backups_controller.rb

```diff
@@ -32,6 +32,9 @@ def status
   end
 
   def create
+    if BackupRestore.backups_disallowed?
```

Comment:
Trivial but this could be one line if you like `raise Discourse::InvalidAccess if BackupRestore.backups_disallowed?`